active/corsair.py

```diff
@@ -0,0 +1,54 @@
+"""
+This is an adaptation of Corsair Scan (https://github.com/Santandersecurityresearch/corsair_scan) as a ZAP active scan script.
+This script will resend requests to all the sites captured in ZAP, injecting different origins. Then, it will read the value of Access-Control-Allow-Origin and based on that, it
+will assess if CORS is properly configured.
+"""
+
+import urlparse
+alertTitle = 'Corsair - CORS Misconfigured.'
+alertDescription = "Cross Origin Resource Sharing (CORS) is misconfigured. \n"
+alertRisk = 2
+alertReliability = 3
+alertSolution = "Configure CORS in a more restrictive way, whitelisting only the sites allowed to access to your domain."
+alertInfo = "Cross-Origin Resource Sharing (CORS) is an HTTP-header based mechanism that allows a server to indicate any other origins (domain, scheme, or port) than its own from which a browser should permit loading of resources."
+cweID = 942
+wascID = 14
+
+SM_ORIGIN = 'https://scarymonster.com'
```

Comment:
Not sure about this domain.

Reply:
it is just an example. which domain would you suggest?

Reply:
`https://example.com`